.github/workflows/build_and_publish_all_images.yaml

```diff
@@ -0,0 +1,227 @@
+name: Build & publish all images
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: The environment for all images, such as "dev" or "master"
+        required: true
+        type: string
+      image_version_1:
+        description: A version for all image tags, such as "foo" in "ghcr.io/user/repo/image:foo"
+        required: true
+        type: string
+      image_version_2:
+        description: Another version for all image tags, such as "bar" in "ghcr.io/user/repo/image:bar"
+        required: false
+        type: string
+
+jobs:
+
+  build_and_publish_app_proxy_image:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./.github/actions/build_and_publish_image
+      with:
+        environment: ${{ inputs.environment }}
+        image_source_directory: images/app-proxy
+        image_name: webrtc-gpu-streaming-app-proxy
```

Comment:
Replace all instances of `webrtc-gpu-streaming-` in image tag basenames with ``. (Remove legacy prefixes).